pkg/api/apikey.go

```diff
@@ -93,7 +93,7 @@ func (hs *HTTPServer) DeleteAPIKey(c *models.ReqContext) response.Response {
 
 // swagger:route POST /auth/keys api_keys addAPIkey
 //
-// Creates an API key.
+// Create an API key.
```

Comment:
this changes is just for demonstrating the case of the Swagger/OpenAPI needing update; but it does not harm to keep it for now